src/Microsoft.Extensions.DependencyInjection.Abstractions/ServiceDescriptor.cs

```diff
@@ -6,84 +6,11 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
-    [DebuggerDisplay("Lifetime = {Lifetime}, ServiceType = {ServiceType}, ImplementationType = {ImplementationType}")]
-    public class ServiceDescriptor
+    [DebuggerDisplay("Lifetime = {Lifetime}, ServiceType = {ServiceType}")]
```

Comment:
I believe so, but it does get overridden by a `DebuggerDisplay` attribute on a derived type.